File: app/src/main/java/com/example/proyectomovil/Models/Actividad.java
package com.example.proyectomovil.Models;

import java.util.ArrayList;
import java.util.List;

public class Actividad {
    public int id_actividad;
    public String nombre;
    public String descripcion;
    public String imagen;
    public String fecha;
    public String favorito;
    public String calificacion;
    public List<Sitio> sitios = new ArrayList<>();
}

File: ProyectoMovilTour/app/Actividad.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Actividad extends Model
{
    protected $table = 'actividad';
    protected $primaryKey = 'id_actividad';

    protected $fillable = [
    	'nombre',
    	'descripcion',
    	'fecha',
    	'imagen',
    ];
    public function sitios()
	{
		$intersectos = IntersectoActividadSitio::all()->where('id_actividad', $this->id_actividad);
		$sitios = [];
		foreach ($intersectos as $intersecto) {
			array_push($sitios, $intersecto->sitio);
		}
		return $sitios;
	}
}
